PostBoard.Api/Controllers/Birthday.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using PostBoard.Api.Data;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace PostBoard.Api.Controllers;
+
+[Route("api/berthday")]
+[ApiController]
+public class BerthdayController : ControllerBase
+{
+    private static List<Birthday> Berthdays = new List<Birthday>();
+
+
+
+    // GET: api/<Berthday>
+    [HttpGet]
+    public IActionResult GetAll()
+    {
+        return Ok(Berthdays);
+    }
+
+    // GET api/<Berthday>/5
+    [HttpGet]
+    [Route("{id:int}")]
+    public IActionResult GetById([FromRoute] int id)
+    {
+        var berthday = Berthdays.FirstOrDefault(d => d.Id == id);
```

Comment:
In lambda expression fix `d` name to `b`, because it should be `Birthday` word's first letter.